app/src/components/datasets/DatasetContentTabs/General/DatasetEntriesTable/TableRow.tsx

```diff
@@ -135,21 +147,19 @@ export const TableRow = ({
 };
 
 const EntrySplit = ({ split }: { split: string }) => {
-  const color = split === DatasetEntrySplit.TRAIN ? "orange.500" : "purple.500";
+  const color = split === DatasetEntrySplit.TRAIN ? "orange" : "purple";
   return (
     <HStack justifyContent="flex-end">
-      <Text
-        fontSize="xs"
-        fontWeight="semibold"
+      <Badge
```

Comment:
Could you add before and after screenshots for this change?

Reply:
![t_vs_b](https://github.com/OpenPipe/OpenPipe/assets/28661894/345a0f2e-7ddc-4c46-a2bc-0509eae5de1b)
Left - Text
Right - Badge

I thought of replacing Text with Badge because Chakra has an exact UI component that looks very similar.

Reply:
Got it! Will include screenshots in the next PR.